AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/ClientBehavior/AWSCognitoAuthPlugin+UserBehavior.swift

```diff
@@ -41,8 +41,10 @@ public extension AWSCognitoAuthPlugin {
         } as! [AuthUserAttributeKey: AuthUpdateAttributeResult]
     }
 
-    func resendConfirmationCode(forUserAttributeKey userAttributeKey: AuthUserAttributeKey,
-                                options: AuthAttributeResendConfirmationCodeRequest.Options? = nil) async throws -> AuthCodeDeliveryDetails {
+    func resendConfirmationCode(
+        forUserAttributeKey userAttributeKey: AuthUserAttributeKey,
+        options: AuthAttributeResendConfirmationCodeRequest.Options? = nil
+    ) async throws -> AuthCodeDeliveryDetails {
```

Comment:
Should we deprecate this method as well? Or not necessary since it's the implementation of the protocol?